shop_profile/tests.py

```
@@ -32,6 +35,15 @@ def setUp(self):
             status=True
         )
         self.client = Client()
+    def test_create_user_no_email(self):
+        with self.assertRaises(ValueError):
+            User.objects.create_user(email="", password=TEST_PASS)
+
+    def test_create_superuser(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_create_superuser`

Add return type annotation: `None`

(ANN201)